09-features.md

```diff
@@ -5,34 +5,36 @@ They are tracked separately since new flags will likely be added over time.
 
 The `features` flags in the routing messages are a subset of the `globalfeatures` flags, since the `localfeatures` are by definition only of interest to direct peers.
 
-Flags are numbered from the least-significant bit at bit 0 (ie. 0x1,
-an even bit).  They are generally assigned in pairs, so that features
-can be introduced as optional (odd bits), and later upgraded to refuse
-old nodes (even bits).  See [BOLT #1: The `init` message](01-messaging.md#the-init-message).
+Flags are numbered from the least-significant bit, at bit 0 (i.e. 0x1,
+an even bit). They are generally assigned in pairs so that features
+can be introduced as optional (odd bits) and later upgraded to be compulsory, refusing
+old nodes (even bits). See [BOLT #1: The `init` Message](01-messaging.md#the-init-message).
 
 ## Assigned `localfeatures` flags
 
 These flags may only be used in the `init` message:
 
-
 | Bits | Name             |Description                                     | Link                                                                |
 |------|------------------|------------------------------------------------|---------------------------------------------------------------------|
-| 0/1  | `option-data-loss-protect` | Requires/supports extra `channel_reestablish` fields | [BOLT #2](02-peer-protocol.md#message-retransmission) |
-| 3  | `initial_routing_sync` | The sending node needs a complete routing information dump | [BOLT #7](07-routing-gossip.md#initial-sync) |
-| 4/5  | `option_upfront_shutdown_script` | Commit to a shutdown scriptpubkey when opening | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
+| 0/1  | `option-data-loss-protect` | Requires or supports extra `channel_reestablish` fields | [BOLT #2](02-peer-protocol.md#message-retransmission) |
+| 3  | `initial_routing_sync` | Indicates that the sending node needs a complete routing information dump | [BOLT #7](07-routing-gossip.md#initial-sync) |
+| 4/5  | `option_upfront_shutdown_script` | Commits to a shutdown scriptpubkey when opening | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 
 ## Assigned `globalfeatures` flags
 
+[ FIXME: Are there currently no globalfeatures flags? If so, we should note that. ]
```

Comment:
Yes, there are none currently defined.  Words here would be good :+1:

Reply:
Will do. I'll sync that back in the morning.